articles/aws-chalice-twilio-whatsapp/index.md

```diff
@@ -0,0 +1,394 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /aws-chalice-twilio-whatsapp/
+title: Sending WhatsApp Messages with Serverless Python Applications using AWS Chalice
+description: Serverless computing enables developers to build software and applications without dealing with servers. It abstracts server management from the responsibilities and worries of a developer. AWS Chalice is a light and fast serverless framework built by AWS. It is a Python-based framework. It leverages the Amazon API Gateway and AWS Lambda. In this article, we set up a Chalice application with the DynamoDB database. We will also integrate Twilio WhatsApp messaging and send messages from our application.
+author: jekayinoluwa-olabemiwo
+date: 2021-06-28T00:00:00-18:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  -url: /engineering-education/aws-chalice-twilio-whatsapp/hero.jpg
+   alt: AWS Chalice example image
+---
+
+
+### Introduction:
+
+Serverless computing enables developers to build software and applications without dealing with servers. It abstracts server management from the responsibilities and worries of a developer. AWS Chalice is a light and fast serverless framework built by AWS. It is a Python-based framework. It leverages the Amazon API Gateway and AWS Lambda.
+
+WhatsApp is a free messaging platform used by over 2 billion people across the world. WhatsApp Messaging API allows developers to build applications for WhatsApp users.
+
+In this article, we set up a Chalice application with the DynamoDB database. We will also integrate Twilio WhatsApp messaging and send messages from our application.
+
+### Prerequisites
+
+- Python 3.6 or a later version
+- venv or virtualenv
```

Comment:
Insert a link to creating virtual environments using virtualenv. You can also mention the pip command to install the library.